src/PKSim.Core/Services/IndividualSimulationEngine.cs

```diff
@@ -95,26 +95,36 @@ private Task runSimulation(IndividualSimulation simulation, bool exportAll, bool
 
          return Task.Run(() =>
          {
-            var modelCoreSimulation = _modelCoreSimulationMapper.MapFrom(simulation, shouldCloneModel:false);
+            var modelCoreSimulation = _modelCoreSimulationMapper.MapFrom(simulation, shouldCloneModel: false);
             var simRunOptions = new SimulationRunOptions
             {
                SimModelExportMode = SimModelExportMode.Optimized,
                CheckForNegativeValues = checkForNegativeValues
             };
-               
+
             var simResults = _simModelManager.RunSimulation(modelCoreSimulation, simRunOptions);
 
             if (!simResults.Success)
                return;
 
             _simulationResultsSynchronizer.Synchronize(simulation, simResults.Results);
+            updateResultsName(simulation);
+
             simulation.ClearPKCache();
 
             if (raiseEvents)
                _eventPublisher.PublishEvent(new SimulationResultsUpdatedEvent(simulation));
          });
       }
 
+      private void updateResultsName(IndividualSimulation simulation)
```

Comment:
I just realize that the name was not consistent depending on WHEN the results was created (brand new simulation or simulation loaded, which also lead to other bugs when renaming a simulation)

That way it is always consistent